README.md

```diff
@@ -57,6 +62,10 @@ then:
 product([f1, ... , fn]): [A1, ... , An] ~> [P1, ... , Pn]
 ```
 
+in short:
+
+We use the operator *product* whenever we want (and we always do if we can) to run multiple fetches in **parallel**.
```

Comment:
This stresses the "performance" argument, I'd rather stress the "I want some things that do not depend one on another". Maybe let's put this somewhere in between?